### DISCUSSION TITLE
Support for Victron MPTT ESPHome (aka Non Victron GX Inverters)

### DISCUSSION THREAD

**Discussion by Roving-Ronin** [slipx06/sunsynk-power-flow-card#365] [Victron]
Hi @slipx06,

Just wondering your thoughts upon adding support for Victron inverters, that use the https://github.com/KinDR007/VictronMPPT-ESPHOME method for integration ?

Working on this for a friend that has a number of remote sites that are offgrid and using Victron SmartSolar MPPT devices (so not the 'smart' Victrons with GX functions) with Solar / Generator to power 48V batteries and communication equipment.

Whilst SunSynk Card currently has 'Victron' added, this is using the https://github.com/sfstar/hass-victron integration and looks to only support GX (i.e. a Smart Range with Modbus + Network connectivity built in), where as these other Victron inverters don't have these functions and the solution is for users to use an ESP board to wire into the SmartSolar MPPT, with the ESP then providing modbus access and network connectivity.

Given the above thinking if a 'Victron MPPT-ESPHome' version should be added to the card, that would allow defining the different 'statusGroups: InverterStatus' messages that this provides, versus the hass-victron / Victron GX with its messages it provides.

My understanding is that to do this would a new sunsynk-power-flow-card/src/inverters/brands/victron-mptt-esphome.ts file, into which the new 'statusGroups: InverterStatus' messages would be defined, whilst the existing image created for the victron.ts could be used for this new 'Victron MPPT-ESPHome' (as the inverters look the same).

Thanks.

**Comment by slipx06** [comment]
Hi @Roving-Ronin 

Unless there are conflicting status codes/messages we can simply add them to the existing array of values. I would prefer to do this than create a new "brand" based on the integration method. The current set of status codes/messages  are displayed below and have not been customised. 

        ```
        standby: {states: ['0', 'standby', 'stand-by'], color: 'blue', message: localize('common.standby')},
        selftest: {states: ['1', 'selftest', 'self-checking'], color: 'yellow', message: localize('common.selftest')},
        normal: {states: ['2', 'normal', 'ok'], color: 'green', message: localize('common.normal')},
        alarm: {states: ['3', 'alarm'], color: 'orange', message: localize('common.alarm')},
        fault: {states: ['4', 'fault'], color: 'red', message: localize('common.fault')},
```
